package.json

```diff
@@ -1,18 +1,19 @@
 {
   "name": "@actions/github-script",
   "description": "A GitHub action for executing a simple script",
-  "version": "7.0.1",
+  "version": "8.0.0",
   "author": "GitHub",
   "license": "MIT",
   "main": "dist/index.js",
+  "type": "module",
```

Comment:
I'm fairly certain this was required when updating Octokit, I don't think it should impact much since this action isn't consumed as a library.

My one concern is that it somehow impacts the custom `require` functionality this action implements, but integration tests should catch that.